enterprise/views.py

```diff
@@ -947,9 +948,8 @@ def get_available_course_modes(self, request, course_run_id, enterprise_catalog)
         """
         modes = EnrollmentApiClient().get_course_modes(course_run_id)
         if not modes:
-            LOGGER.warning('Unable to get course modes for course run id {course_run_id}.'.format(
-                course_run_id=course_run_id
-            ))
+            LOGGER.warning('[Enterprise Enrollment] Unable to get course modes. '
+                           'CourseRun: {course_run_id}'.format(course_run_id=course_run_id))
```

Comment:
Should we use `Course` here instead of `CourseRun` since course run id is a course id actually. This might help filter logs a bit easier ?

Reply:
Course and CourseRun are logged separately elsewhere in the file also. Hence following the same convention.